### ISSUE TITLE
Error on call denops_shared_server#install()

### ISSUE THREAD

**Issue by yukimemi** [vim-denops/denops-shared-server.vim#9]
Error on call denops_shared_server#install()

```vim
function denops_shared_server#install の処理中にエラーが検出されました:
行    6:
E117: 未知の関数です: denops#util#script_path
行   15:
E121: Undefined variable: options
E116: 関数の無効な引数です: denops_shared_server#launchctl#install
行   22:
E117: 未知の関数です: denops#util#info
```

**Comment by lambdalisue** [comment]
Those functions are removed from v4.0.0 due to deprecation (https://github.com/vim-denops/denops.vim/pull/207) so denops-shared-server.vim must be fixed to support v4.0.0

**Comment by lambdalisue** [comment]
Fixed by https://github.com/vim-denops/denops-shared-server.vim/pull/10
